lib/db/memory.js

```diff
@@ -57,6 +60,16 @@ const unique = require('../unique');
  *       userId: <user_id>,
  *       type: <string>,
  *       scope: <string>,
+ *       createdAt: <timestamp>,
+ *       expiresAt: <timestamp>
+ *     }
+ *   },
+ *   refreshTokens: {
+ *     <token>: {
+ *       token: <string>,
+ *       clientId: <client_id>,
+ *       userId: <user_id>,
+ *       scope: <string>,
  *       createdAt: <timestamp>
```

Comment:
It probably would be useful, yes. Those that ignore the refresh token could be inadvertently generating new ones every time, and that would load up the table with a bunch of refresh tokens that sit there forever, ready to be guessed?